\toc 

# üìÑ Curriculum Vitae

## üéì Education

- **April 2010 -- March 2015**: **Doctor of Philosophy (PhD)**, *Mention in Basic and Applied Sciences*, Universidad Nacional de Quilmes, Buenos Aires, Argentina
  - **Title:** Molecular evolution and conformational diversity in proteins
  - **Supervisors:** Gustavo Parisi and Cristina Marino-Buslje.
- **2003 -- 2009**: Licentiate degree in **Biotechnology**, Orientation to Genetics and Molecular Biology, Universidad Nacional de Quilmes, Buenos Aires, Argentina.
- **2003 -- 2006**: Bachelor of Technology, Laboratory Technician, Universidad Nacional de Quilmes, Buenos Aires, Argentina.

## üíº Experience

### üë®‚Äçüíª Research

- **September 2022**: Associate professor *(Ma√Ætre de conf√©rences)*, Universit√© Paris-Saclay, I2BC, Gif-sur-Yvette, France.
- **15 April 2022 -- August 31, 2022**: Postdoctoral Fellow, UMR 7205 CNRS, ISYEB, Mus√©um national d'Histoire naturelle, Paris, France. I worked on modeling the evolution of protein structures under the supervision of Olivier Gascuel.
- **March 2018 -- September 30, 2020**: Postdoctoral Fellow, UMR 7238 CNRS "Biologie Computationnelle et Quantitative", Sorbonne Universit√©, Paris, France. Research focused on the evolution of alternative splicing and the development of tools for identifying homologous exons, conducted under the supervision of Elodie Laine and Hugues Richard.
- **January 2018 -- March 2018**: Postdoctoral Fellow, UMR 7205 ISYEB, "AIRE", Sorbonne Universit√©, Paris, France. Conducted studies on the evolution of domain architectures through network analysis, under the supervision of Eric Bapteste.
- **September 2017 -- January 2018**: Postdoctoral Fellow (short-term research stay), Sorbonne Universit√©, Paris, France. Focused on the evolution of domain architectures through network analysis, under the supervision of Cristina Marino-Buslje and Eric Bapteste.
- **April 2015 -- January 2018**: Postdoctoral Fellow, CONICET, Leloir Institute Foundation, Buenos Aires, Argentina. I developed tools integrating coevolution and structural data, supervised by Cristina Marino-Buslje.
- **July 2013 -- September 2013**: PhD student (short-term research stay), Universit√† degli Studi di Padova, Padova, Italy. I studied protein disorder as another aspect of conformational diversity, supervised by Silvio Tosatto and Gustavo Parisi.
- **April 2010 -- March 2015**: PhD student, CONICET, Universidad Nacional de Quilmes, Buenos Aires, Argentina. Researched the relationship between protein conformational diversity and molecular evolution, supervised by Gustavo Parisi and Cristina Marino-Buslje.

### üë®‚Äçüè´ Teaching

- **September 2022**: Associate professor *(Ma√Ætre de conf√©rences)*, Universit√© Paris-Saclay, Gif-sur-Yvette, France.
- **March 2020**: Temporary teacher *(Enseignant vacataire)*, "Advanced Programming", Station Biologique de Roscoff UFR937, Sorbonne Universit√©, Paris, France.
- **February 2019 -- March 2019**: Temporary teacher *(Enseignant vacataire)*, "Introduction to Bioinformatics", Sorbonne Universit√©, Paris, France.
- **April 2017 -- May 2017**: Adjunct Professor, MSc Bioinformatics and Systems Biology course: "Categorical Data Analysis", Universidad Nacional de Quilmes, Buenos Aires, Argentina.
- **August 2010 -- May 2017**: Assistant Professor, Department of Physics, Universidad Nacional de Quilmes, Buenos Aires, Argentina.
- **June 2016 -- August 2016**: Adjunct Professor, MSc Bioinformatics and Systems Biology course: "Categorical Data Analysis", Universidad Nacional del Noroeste de la Provincia de Buenos Aires, Buenos Aires, Argentina.
- **August 2014 -- December 2014**: Adjunct Professor, Agrobiotechnology Engineering course: "Bioinformatics", Universidad Nacional de General San Mart√≠n, Buenos Aires, Argentina.
- **March 2009 -- December 2009**: Student Assistant, Department of Physics, Universidad Nacional de Quilmes, Buenos Aires, Argentina.
- **March 2007 -- December 2007**: Student Assistant, Department of Mathematics, Universidad Nacional de Quilmes, Buenos Aires, Argentina.

## üìù Publications

### üìò Books

- Zea, D. J. (2022). **Interactive Visualization and Plotting with Julia: Create impressive data visualizations through Julia packages such as Plots, Makie, Gadfly, and more.** Packt Publishing Ltd. [*link*](https://packtpublishing.github.io/Interactive-Visualization-and-Plotting-with-Julia/)

- Banchero, M., Bustamante, J. P., Lanzarotti, E., Revuelta, M. V., Teppa, E., Zea, D. J., & Stocchi, N. (2022). **Vida.exe: Desaf√≠os y aventuras de la bioinform√°tica.** Fondo de Cultura Econ√≥mica Argentina. [*link*](https://books.google.fr/books?id=MSmbEAAAQBAJ)


### üîé Research Articles

- Bret, H., Gao, J., Zea, D. J., Andreani, J., & Guerois, R. (2024). **From interaction networks to interfaces, scanning intrinsically disordered regions using AlphaFold2.** Nature Communications, 15(1), 597. [*link*](https://doi.org/10.1038/s41467-023-44288-7)

- Szatkownik, A., Zea, D. J., Richard, H., & Laine, E. (2023). **Building alternative splicing and evolution-aware sequence-structure maps for protein repeats.** Journal of Structural Biology, Volume 215, Issue 3. [*link*](https://doi.org/10.1016/j.jsb.2023.107997)

- Zea, D. J., Mac Donagh, J., Benitez, G., Donadio, C. G., Marchetti, J., Palopoli, N., Fornasari, M. S., & Parisi, G. (2022). **Evolutionary rates in human amyloid proteins reveal their intrinsic metastability.** bioRxiv, 2022‚Äì09. [*link*](https://doi.org/10.1101/2022.09.07.506994)

- Zea, D. J., Richard, H., & Laine, E. (2022). **ASES: visualizing evolutionary conservation of alternative splicing in proteins.** Bioinformatics, 38(9), 2615‚Äì2616. [*link*](https://doi.org/10.1093/bioinformatics/btac105)

- Salda√±o, T., Escobedo, N., Marchetti, J., Zea, D. J., Mac Donagh, J., Velez Rueda, A. J., Gonik, E., Garcia, A. M., Salas, M., Peters, T. & Demitroff, N. (2022). **Impact of protein conformational diversity on AlphaFold2 predictions.** Bioinformatics [*link*](https://doi.org/10.1093/bioinformatics/btac202)

- Zea, D. J., Laskina, S., Baudin, A., Richard, H., & Laine, E. (2021). **Assessing conservation of alternative splicing with evolutionary splicing graphs.** Genome Research, gr-274696. [*link*](https://doi.org/10.1093/bioinformatics/btac105)

- Palopoli, N., Marchetti, J., Monzon, A. M., Zea, D. J., Tosatto, S. C. E., Fornasari, M. S. & Parisi, G. (2020). **Intrinsically Disordered Protein Ensembles Shape Evolutionary Rates Revealing Conformational Patterns.** Journal of Molecular Biology  [*link*](https://doi.org/10.1016/j.sbi.2015.02.005)

- Teppa, E., Zea, D. J., Oteri, F. & Carbone, A. (2020). **COVTree: Coevolution in OVerlapped sequences by Tree analysis server.** *Nucleic Acids Research, gkaa330* [*link*](https://doi.org/10.1093/nar/gkaa330)

- Teppa, E., Nadalin, F., Combet, C., Zea, D. J., David, L. & Carbone, A. (2020). **Coevolution analysis of amino-acids reveals diversified drug resistance solutions in viral sequences: the case study of Hepatitis B Virus.** *Virus Evolution, 6(1), veaa006* [*link*](https://doi.org/10.1093/ve/veaa006)

- Ait-hamlat, A., Zea, D. J., Labeeuw, A., Polit, L., Richard, H. & Laine, E. (2020). **Transcripts' evolutionary history and structural dynamics give mechanistic insights into the functional diversity of the JNK family.** *Journal of Molecular Biology* [*link*](https://doi.org/10.1016/j.jmb.2020.01.032)

- Ochoa, S., Mart√≠nez‚ÄêP√©rez, E., Zea, D. J., Molina‚ÄêVila, M. A., & Marino‚ÄêBuslje, C. (2019). **Comutation and exclusion analysis in human tumors: A tool for cancer biology studies and for rational selection of multitargeted therapeutic approaches.** *Human mutation, 40(4), 413-425.* [*link*](https://doi.org/10.1002/humu.23705)

- Zea, D. J., Monzon, A. M., Parisi, G., & Marino-Buslje, C. (2018). **How is structural divergence related to evolutionary information?.** *Molecular phylogenetics and evolution*, 127, 859-866. [*link*](https://www.sciencedirect.com/science/article/pii/S1055790317306188)

- Marino-Buslje, C., Monzon, A. M., Zea, D. J., Fornasari, M. S., & Parisi, G. (2017). **On the dynamical incompleteness of the Protein Data Bank.** *Briefings in Bioinformatics* [*link*](https://doi.org/10.1093/bib/bbx084)

- Teppa, E., Zea, D. J., & Marino‚ÄêBuslje, C. (2017). **Protein‚Äìprotein interactions leave evolutionary footprints: High molecular coevolution at the core of interfaces.** *Protein Science*, 26(12), 2438-2444. [*link*](https://doi.org/10.1002/pro.3318)

- Monzon, A. M., Zea, D. J., Marino-Buslje, C., & Parisi, G. (2017). **Homology modeling in a dynamical world.** *Protein Science*, 26(11), 2195-2206. [*link*](https://doi.org/10.1002/pro.3274)

- Monzon, A. M., Zea, D. J., Fornasari, M. S., Salda√±o, T., Fernandez-Alberti, S., Tosatto, S. C., & Parisi, G. (2017). **Conformational diversity analysis reveals three functional mechanisms in proteins.** *PLOS Computational Biology*, 13, no. 2: e1005398. [*link*](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005398)  

- Zea, D. J., Anfossi, D., Nielsen, M., & Marino-Buslje, C. (2016). **MIToS. jl: mutual information tools for protein sequence analysis in the Julia language.** *Bioinformatics*, btw646. [*link*](https://academic.oup.com/bioinformatics/article-abstract/doi/10.1093/bioinformatics/btw646/2608634/MIToS-jl-mutual-information-tools-for-protein)

- Zea, D. J., Monzon, A. M., Gonzalez, C., Fornasari, M. S., Tosatto, S. C., & Parisi, G. (2016). **Disorder transitions and conformational diversity cooperatively modulate biological function in proteins.** *Protein Science*, 25(6), 1138-1146. [*link*](http://onlinelibrary.wiley.com/doi/10.1002/pro.2931/full)

- Iserte, J., Simonetti, F. L., Zea, D. J., Teppa, E., & Marino-Buslje, C. (2015). **I-COMS: interprotein-COrrelated mutations server.**  *Nucleic acids research*, 43(W1), W320-W325. [*link*](https://academic.oup.com/nar/article/43/W1/W320/2468009/I-COMS-Interprotein-COrrelated-Mutations-Server)

- Parisi, G., Zea, D. J., Monzon, A. M., & Marino-Buslje, C. (2015). **Conformational diversity and the emergence of sequence signatures during evolution.** *Current opinion in structural biology*, 32, 58-65. [*link*](http://www.sciencedirect.com/science/article/pii/S0959440X15000147)

- Zea, D. J., Monzon, A. M., Fornasari, M. S., Marino-Buslje, C., & Parisi, G. (2013). **Protein conformational diversity correlates with evolutionary rate.** *Molecular biology and evolution*, 30(7), 1500-1503. [*link*](http://mbe.oxfordjournals.org/content/30/7/1500.full)

### üìÑ Other Articles

- Parra, R. G., Defelipe, L. A., Guzovsky, A. B., Monzon, A. M., Cravero, F., Mancini, E., ... & Gonzalez, N. N. (2016). **Highlights of the 1st Argentine Symposium of Young Bioinformatics Researchers (1SAJIB) organized by the ISCB RSG-Argentina.** *PeerJ Preprints*, 4, e2494v1.  [*link*](https://peerj.com/preprints/2494/)

### üìö Book chapters

- Zea, D. J., Teppa, E., & Marino-Buslje, C. (2023). **Easy Not Easy: Comparative Modeling with High-Sequence Identity Templates.** In Homology Modeling: Methods and Protocols (pp. 83‚Äì100). Springer US New York, NY. [*link*](https://link.springer.com/protocol/10.1007/978-1-0716-2974-1_5)

- Teppa, E., Zea, D. J., & Buslje, C. M.  (2012). **Identification of Coevolving Amino Acids using Mutual Information.** *iConcept Press*. [*link*](https://www.researchgate.net/profile/Elin_Teppa/publication/261995698_Identification_of_Coevolving_Amino_Acids_using_Mutual_Information/links/00b4953631b3666180000000.pdf)

- Monzon, A. M., Fornasari, M. S., Zea, D. J., & Parisi, G. (2019). **Exploring Protein Conformational Diversity.** *In Computational Methods in Protein Evolution* (pp. 353-365). Humana Press, New York, NY. [*link*](https://link.springer.com/protocol/10.1007/978-1-4939-8736-8_20)
