[Google Scholar](https://scholar.google.com.ar/citations?user=Z4sqaB0AAAAJ&hl=en&oi=ao) | [ORCID](https://orcid.org/0000-0002-4254-9320) | [Scopus](https://www.scopus.com/authid/detail.uri?authorId=56534065600) | [GitHub](https://github.com/diegozea)

**Associate Professor** at *Université Paris-Saclay*. I am interested in protein structure, interactions and evolution.

# Education

<dl>
  <dt>April 2010 - March 2015</dt>
  <dd><strong>Doctor of Philosophy (PhD)</strong>, <em>Mention in Basic and Applied Sciences</em>. Universidad Nacional de Quilmes, Buenos Aires, Argentina.</dd>
  <dt></dt>
  <dd>
    <dl>
    <dt>Thesis title</dt><dd><em>Molecular evolution and conformational diversity in proteins</em></dd>
      <dt>Field of study</dt><dd>Bioinformatics</dd>
      <dt>Supervisor</dt><dd>PhD Gustavo Parisi</dd>
      <dt>Co-supervisor</dt><dd>MS, PhD Cristina Marino-Buslje</dd>
    </dl>
  </dd>
  <dt>2003 - 2009</dt>
  <dd><strong>Licentiate degree in Biotechnology</strong>, <em>Orientation to Genetics and Molecular Biology</em>. Universidad Nacional de Quilmes, Buenos Aires, Argentina</dd>
  <dt>2003 - 2006</dt>
  <dd><strong>Bachelor of Technology</strong>, <em>Laboratory Technician</em>. Universidad Nacional de Quilmes, Buenos Aires, Argentina.</dd>
</dl>

# Publications

### Book

- [Interactive Visualization and Plotting with Julia](https://packtpublishing.github.io/Interactive-Visualization-and-Plotting-with-Julia/)

### Research Articles

- Saldaño, T., Escobedo, N., Marchetti, J., Zea, D. J., Mac Donagh, J., Velez Rueda, A. J., Gonik, E., Garcia, A. M., Salas, M., Peters, T. & Demitroff, N. (2022). **Impact of protein conformational diversity on AlphaFold2 predictions.** Bioinformatics [*link*](https://doi.org/10.1093/bioinformatics/btac202)

- Zea, D. J., Laskina, S., Baudin, A., Richard, H., & Laine, E. (2021). **Assessing conservation of alternative splicing with evolutionary splicing graphs.** Genome Research, gr-274696. [*link*](https://doi.org/10.1093/bioinformatics/btac105)

- Palopoli, N., Marchetti, J., Monzon, A. M., Zea, D. J., Tosatto, S. C. E., Fornasari, M. S. & Parisi, G. (2020). **Intrinsically Disordered Protein Ensembles Shape Evolutionary Rates Revealing Conformational Patterns.** Journal of Molecular Biology  [*link*](https://doi.org/10.1016/j.sbi.2015.02.005)

- Teppa, E., Zea, D. J., Oteri, F. & Carbone, A. (2020). **COVTree: Coevolution in OVerlapped sequences by Tree analysis server.** *Nucleic Acids Research, gkaa330* [*link*](https://doi.org/10.1093/nar/gkaa330)

- Teppa, E., Nadalin, F., Combet, C., Zea, D. J., David, L. & Carbone, A. (2020). **Coevolution analysis of amino-acids reveals diversified drug resistance solutions in viral sequences: the case study of Hepatitis B Virus.** *Virus Evolution, 6(1), veaa006* [*link*](https://doi.org/10.1093/ve/veaa006)

- Ait-hamlat, A., Zea, D. J., Labeeuw, A., Polit, L., Richard, H. & Laine, E. (2020). **Transcripts' evolutionary history and structural dynamics give mechanistic insights into the functional diversity of the JNK family.** *Journal of Molecular Biology* [*link*](https://doi.org/10.1016/j.jmb.2020.01.032)

- Ochoa, S., Martínez‐Pérez, E., Zea, D. J., Molina‐Vila, M. A., & Marino‐Buslje, C. (2019). **Comutation and exclusion analysis in human tumors: A tool for cancer biology studies and for rational selection of multitargeted therapeutic approaches.** *Human mutation, 40(4), 413-425.* [*link*](https://doi.org/10.1002/humu.23705)

- Zea, D. J., Monzon, A. M., Parisi, G., & Marino-Buslje, C. (2018). **How is structural divergence related to evolutionary information?.** *Molecular phylogenetics and evolution*, 127, 859-866. [*link*](https://www.sciencedirect.com/science/article/pii/S1055790317306188)

- Marino-Buslje, C., Monzon, A. M., Zea, D. J., Fornasari, M. S., & Parisi, G. (2017). **On the dynamical incompleteness of the Protein Data Bank.** *Briefings in Bioinformatics* [*link*](https://doi.org/10.1093/bib/bbx084)

- Teppa, E., Zea, D. J., & Marino‐Buslje, C. (2017). **Protein–protein interactions leave evolutionary footprints: High molecular coevolution at the core of interfaces.** *Protein Science*, 26(12), 2438-2444. [*link*](https://doi.org/10.1002/pro.3318)

- Monzon, A. M., Zea, D. J., Marino-Buslje, C., & Parisi, G. (2017). **Homology modeling in a dynamical world.** *Protein Science*, 26(11), 2195-2206. [*link*](https://doi.org/10.1002/pro.3274)

- Monzon, A. M., Zea, D. J., Fornasari, M. S., Saldaño, T., Fernandez-Alberti, S., Tosatto, S. C., & Parisi, G. (2017). **Conformational diversity analysis reveals three functional mechanisms in proteins.** *PLOS Computational Biology*, 13, no. 2: e1005398. [*link*](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005398)  

- Zea, D. J., Anfossi, D., Nielsen, M., & Marino-Buslje, C. (2016). **MIToS. jl: mutual information tools for protein sequence analysis in the Julia language.** *Bioinformatics*, btw646. [*link*](https://academic.oup.com/bioinformatics/article-abstract/doi/10.1093/bioinformatics/btw646/2608634/MIToS-jl-mutual-information-tools-for-protein)

- Zea, D. J., Monzon, A. M., Gonzalez, C., Fornasari, M. S., Tosatto, S. C., & Parisi, G. (2016). **Disorder transitions and conformational diversity cooperatively modulate biological function in proteins.** *Protein Science*, 25(6), 1138-1146. [*link*](http://onlinelibrary.wiley.com/doi/10.1002/pro.2931/full)

- Iserte, J., Simonetti, F. L., Zea, D. J., Teppa, E., & Marino-Buslje, C. (2015). **I-COMS: interprotein-COrrelated mutations server.**  *Nucleic acids research*, 43(W1), W320-W325. [*link*](https://academic.oup.com/nar/article/43/W1/W320/2468009/I-COMS-Interprotein-COrrelated-Mutations-Server)

- Parisi, G., Zea, D. J., Monzon, A. M., & Marino-Buslje, C. (2015). **Conformational diversity and the emergence of sequence signatures during evolution.** *Current opinion in structural biology*, 32, 58-65. [*link*](http://www.sciencedirect.com/science/article/pii/S0959440X15000147)

- Zea, D. J., Monzon, A. M., Fornasari, M. S., Marino-Buslje, C., & Parisi, G. (2013). **Protein conformational diversity correlates with evolutionary rate.** *Molecular biology and evolution*, 30(7), 1500-1503. [*link*](http://mbe.oxfordjournals.org/content/30/7/1500.full)

### Other Articles

- Parra, R. G., Defelipe, L. A., Guzovsky, A. B., Monzon, A. M., Cravero, F., Mancini, E., ... & Gonzalez, N. N. (2016). **Highlights of the 1st Argentine Symposium of Young Bioinformatics Researchers (1SAJIB) organized by the ISCB RSG-Argentina.** *PeerJ Preprints*, 4, e2494v1.  [*link*](https://peerj.com/preprints/2494/)

### Book chapters

- Teppa, E., Zea, D. J., & Buslje, C. M.  (2012). **Identification of Coevolving Amino Acids using Mutual Information.** *iConcept Press*. [*link*](https://www.researchgate.net/profile/Elin_Teppa/publication/261995698_Identification_of_Coevolving_Amino_Acids_using_Mutual_Information/links/00b4953631b3666180000000.pdf)

- Monzon, A. M., Fornasari, M. S., Zea, D. J., & Parisi, G. (2019). **Exploring Protein Conformational Diversity.** *In Computational Methods in Protein Evolution* (pp. 353-365). Humana Press, New York, NY. [*link*](https://link.springer.com/protocol/10.1007/978-1-4939-8736-8_20)
