<!doctype html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  
  <!-- bootstrap@5.3.1 and bootstrap icon@1.10-->
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
<link rel="stylesheet" href="/css/style.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3">
<!-- fonts -->
<link data-n-head="ssr" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
<!-- favicon generated through https://realfavicongenerator.net/-->
<link rel="apple-touch-icon" sizes="180x180" href="/assets/icon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/assets/icon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/assets/icon/favicon-16x16.png">
<link rel="manifest" href="/assets/icon/site.webmanifest">
<link rel="mask-icon" href="/assets/icon/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" content="#ffffff">

   <title>üìÑ Curriculum Vitae</title>  
</head>
<body>
  <header data-bs-theme="dark">
  <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
    <div class="container-fluid">
      <a href="/" class="navbar-brand d-flex align-items-center">
        Diego Javier Zea
      </a>
      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse" id="navbarCollapse">
        <ul class="navbar-nav me-auto mb-2 mb-md-0">
          <!--
          <li class="nav-item">
            <a class="nav-link" href="/blog/">Blog</a>
          </li>
          -->
          <!--
          <li class="nav-item">
            <a class="nav-link" href="/software/">Software</a>
          </li>
          -->
          <li class="nav-item">
            <a class="nav-link" href="/cv">CV</a>
          </li>
        </ul>
        <!--
        <form class="d-flex" role="search">
          <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
          <button class="btn btn-outline-success" type="submit">Search</button>
        </form>
        -->
      </div>
    </div>
  </nav>
</header>
  <div class="container py-3 px-3 mx-auto"><div class="franklin-content"><p><div class="franklin-toc"><ol><li><a href="#education">üéì Education</a></li><li><a href="#experience">üíº Experience</a><ol><li><a href="#research">üë®‚Äçüíª Research</a></li><li><a href="#teaching">üë®‚Äçüè´ Teaching</a></li></ol></li><li><a href="#publications">üìù Publications</a><ol><li><a href="#books">üìò Books</a></li><li><a href="#research_articles">üîé Research Articles</a></li><li><a href="#other_articles">üìÑ Other Articles</a></li><li><a href="#book_chapters">üìö Book chapters</a></li></ol></li></ol></div> </p>
<h1 id="curriculum_vitae"><a href="#curriculum_vitae" class="header-anchor">üìÑ Curriculum Vitae</a></h1>
<h2 id="education"><a href="#education" class="header-anchor">üéì Education</a></h2>
<ul>
<li><p><strong>April 2010 ‚Äì March 2015</strong>: <strong>Doctor of Philosophy &#40;PhD&#41;</strong>, <em>Mention in Basic and Applied Sciences</em>, Universidad Nacional de Quilmes, Buenos Aires, Argentina</p>
<ul>
<li><p><strong>Title:</strong> Molecular evolution and conformational diversity in proteins</p>
</li>
<li><p><strong>Supervisors:</strong> Gustavo Parisi and Cristina Marino-Buslje.</p>
</li>
</ul>
</li>
<li><p><strong>2003 ‚Äì 2009</strong>: Licentiate degree in <strong>Biotechnology</strong>, Orientation to Genetics and Molecular Biology, Universidad Nacional de Quilmes, Buenos Aires, Argentina.</p>
</li>
<li><p><strong>2003 ‚Äì 2006</strong>: Bachelor of Technology, Laboratory Technician, Universidad Nacional de Quilmes, Buenos Aires, Argentina.</p>
</li>
</ul>
<h2 id="experience"><a href="#experience" class="header-anchor">üíº Experience</a></h2>
<h3 id="research"><a href="#research" class="header-anchor">üë®‚Äçüíª Research</a></h3>
<ul>
<li><p><strong>September 2022</strong>: Associate professor <em>&#40;Ma√Ætre de conf√©rences&#41;</em>, Universit√© Paris-Saclay, I2BC, Gif-sur-Yvette, France.</p>
</li>
<li><p><strong>15 April 2022 ‚Äì August 31, 2022</strong>: Postdoctoral Fellow, UMR 7205 CNRS, ISYEB, Mus√©um national d&#39;Histoire naturelle, Paris, France. I worked on modeling the evolution of protein structures under the supervision of Olivier Gascuel.</p>
</li>
<li><p><strong>March 2018 ‚Äì September 30, 2020</strong>: Postdoctoral Fellow, UMR 7238 CNRS &quot;Biologie Computationnelle et Quantitative&quot;, Sorbonne Universit√©, Paris, France. Research focused on the evolution of alternative splicing and the development of tools for identifying homologous exons, conducted under the supervision of Elodie Laine and Hugues Richard.</p>
</li>
<li><p><strong>January 2018 ‚Äì March 2018</strong>: Postdoctoral Fellow, UMR 7205 ISYEB, &quot;AIRE&quot;, Sorbonne Universit√©, Paris, France. Conducted studies on the evolution of domain architectures through network analysis, under the supervision of Eric Bapteste.</p>
</li>
<li><p><strong>September 2017 ‚Äì January 2018</strong>: Postdoctoral Fellow &#40;short-term research stay&#41;, Sorbonne Universit√©, Paris, France. Focused on the evolution of domain architectures through network analysis, under the supervision of Cristina Marino-Buslje and Eric Bapteste.</p>
</li>
<li><p><strong>April 2015 ‚Äì January 2018</strong>: Postdoctoral Fellow, CONICET, Leloir Institute Foundation, Buenos Aires, Argentina. I developed tools integrating coevolution and structural data, supervised by Cristina Marino-Buslje.</p>
</li>
<li><p><strong>July 2013 ‚Äì September 2013</strong>: PhD student &#40;short-term research stay&#41;, Universit√† degli Studi di Padova, Padova, Italy. I studied protein disorder as another aspect of conformational diversity, supervised by Silvio Tosatto and Gustavo Parisi.</p>
</li>
<li><p><strong>April 2010 ‚Äì March 2015</strong>: PhD student, CONICET, Universidad Nacional de Quilmes, Buenos Aires, Argentina. Researched the relationship between protein conformational diversity and molecular evolution, supervised by Gustavo Parisi and Cristina Marino-Buslje.</p>
</li>
</ul>
<h3 id="teaching"><a href="#teaching" class="header-anchor">üë®‚Äçüè´ Teaching</a></h3>
<ul>
<li><p><strong>September 2022</strong>: Associate professor <em>&#40;Ma√Ætre de conf√©rences&#41;</em>, Universit√© Paris-Saclay, Gif-sur-Yvette, France.</p>
</li>
<li><p><strong>March 2020</strong>: Temporary teacher <em>&#40;Enseignant vacataire&#41;</em>, &quot;Advanced Programming&quot;, Station Biologique de Roscoff UFR937, Sorbonne Universit√©, Paris, France.</p>
</li>
<li><p><strong>February 2019 ‚Äì March 2019</strong>: Temporary teacher <em>&#40;Enseignant vacataire&#41;</em>, &quot;Introduction to Bioinformatics&quot;, Sorbonne Universit√©, Paris, France.</p>
</li>
<li><p><strong>April 2017 ‚Äì May 2017</strong>: Adjunct Professor, MSc Bioinformatics and Systems Biology course: &quot;Categorical Data Analysis&quot;, Universidad Nacional de Quilmes, Buenos Aires, Argentina.</p>
</li>
<li><p><strong>August 2010 ‚Äì May 2017</strong>: Assistant Professor, Department of Physics, Universidad Nacional de Quilmes, Buenos Aires, Argentina.</p>
</li>
<li><p><strong>June 2016 ‚Äì August 2016</strong>: Adjunct Professor, MSc Bioinformatics and Systems Biology course: &quot;Categorical Data Analysis&quot;, Universidad Nacional del Noroeste de la Provincia de Buenos Aires, Buenos Aires, Argentina.</p>
</li>
<li><p><strong>August 2014 ‚Äì December 2014</strong>: Adjunct Professor, Agrobiotechnology Engineering course: &quot;Bioinformatics&quot;, Universidad Nacional de General San Mart√≠n, Buenos Aires, Argentina.</p>
</li>
<li><p><strong>March 2009 ‚Äì December 2009</strong>: Student Assistant, Department of Physics, Universidad Nacional de Quilmes, Buenos Aires, Argentina.</p>
</li>
<li><p><strong>March 2007 ‚Äì December 2007</strong>: Student Assistant, Department of Mathematics, Universidad Nacional de Quilmes, Buenos Aires, Argentina.</p>
</li>
</ul>
<h2 id="publications"><a href="#publications" class="header-anchor">üìù Publications</a></h2>
<h3 id="books"><a href="#books" class="header-anchor">üìò Books</a></h3>
<ul>
<li><p>Zea, D. J. &#40;2022&#41;. <strong>Interactive Visualization and Plotting with Julia: Create impressive data visualizations through Julia packages such as Plots, Makie, Gadfly, and more.</strong> Packt Publishing Ltd. <a href="https://packtpublishing.github.io/Interactive-Visualization-and-Plotting-with-Julia/"><em>link</em></a></p>
</li>
<li><p>Banchero, M., Bustamante, J. P., Lanzarotti, E., Revuelta, M. V., Teppa, E., Zea, D. J., &amp; Stocchi, N. &#40;2022&#41;. <strong>Vida.exe: Desaf√≠os y aventuras de la bioinform√°tica.</strong> Fondo de Cultura Econ√≥mica Argentina. <a href="https://books.google.fr/books?id&#61;MSmbEAAAQBAJ"><em>link</em></a></p>
</li>
</ul>
<h3 id="research_articles"><a href="#research_articles" class="header-anchor">üîé Research Articles</a></h3>
<ul>
<li><p>Bret, H., Gao, J., Zea, D. J., Andreani, J., &amp; Guerois, R. &#40;2024&#41;. <strong>From interaction networks to interfaces, scanning intrinsically disordered regions using AlphaFold2.</strong> Nature Communications, 15&#40;1&#41;, 597. <a href="https://doi.org/10.1038/s41467-023-44288-7"><em>link</em></a></p>
</li>
<li><p>Szatkownik, A., Zea, D. J., Richard, H., &amp; Laine, E. &#40;2023&#41;. <strong>Building alternative splicing and evolution-aware sequence-structure maps for protein repeats.</strong> Journal of Structural Biology, Volume 215, Issue 3. <a href="https://doi.org/10.1016/j.jsb.2023.107997"><em>link</em></a></p>
</li>
<li><p>Zea, D. J., Mac Donagh, J., Benitez, G., Donadio, C. G., Marchetti, J., Palopoli, N., Fornasari, M. S., &amp; Parisi, G. &#40;2022&#41;. <strong>Evolutionary rates in human amyloid proteins reveal their intrinsic metastability.</strong> bioRxiv, 2022‚Äì09. <a href="https://doi.org/10.1101/2022.09.07.506994"><em>link</em></a></p>
</li>
<li><p>Zea, D. J., Richard, H., &amp; Laine, E. &#40;2022&#41;. <strong>ASES: visualizing evolutionary conservation of alternative splicing in proteins.</strong> Bioinformatics, 38&#40;9&#41;, 2615‚Äì2616. <a href="https://doi.org/10.1093/bioinformatics/btac105"><em>link</em></a></p>
</li>
<li><p>Salda√±o, T., Escobedo, N., Marchetti, J., Zea, D. J., Mac Donagh, J., Velez Rueda, A. J., Gonik, E., Garcia, A. M., Salas, M., Peters, T. &amp; Demitroff, N. &#40;2022&#41;. <strong>Impact of protein conformational diversity on AlphaFold2 predictions.</strong> Bioinformatics <a href="https://doi.org/10.1093/bioinformatics/btac202"><em>link</em></a></p>
</li>
<li><p>Zea, D. J., Laskina, S., Baudin, A., Richard, H., &amp; Laine, E. &#40;2021&#41;. <strong>Assessing conservation of alternative splicing with evolutionary splicing graphs.</strong> Genome Research, gr-274696. <a href="https://doi.org/10.1093/bioinformatics/btac105"><em>link</em></a></p>
</li>
<li><p>Palopoli, N., Marchetti, J., Monzon, A. M., Zea, D. J., Tosatto, S. C. E., Fornasari, M. S. &amp; Parisi, G. &#40;2020&#41;. <strong>Intrinsically Disordered Protein Ensembles Shape Evolutionary Rates Revealing Conformational Patterns.</strong> Journal of Molecular Biology  <a href="https://doi.org/10.1016/j.sbi.2015.02.005"><em>link</em></a></p>
</li>
<li><p>Teppa, E., Zea, D. J., Oteri, F. &amp; Carbone, A. &#40;2020&#41;. <strong>COVTree: Coevolution in OVerlapped sequences by Tree analysis server.</strong> <em>Nucleic Acids Research, gkaa330</em> <a href="https://doi.org/10.1093/nar/gkaa330"><em>link</em></a></p>
</li>
<li><p>Teppa, E., Nadalin, F., Combet, C., Zea, D. J., David, L. &amp; Carbone, A. &#40;2020&#41;. <strong>Coevolution analysis of amino-acids reveals diversified drug resistance solutions in viral sequences: the case study of Hepatitis B Virus.</strong> <em>Virus Evolution, 6&#40;1&#41;, veaa006</em> <a href="https://doi.org/10.1093/ve/veaa006"><em>link</em></a></p>
</li>
<li><p>Ait-hamlat, A., Zea, D. J., Labeeuw, A., Polit, L., Richard, H. &amp; Laine, E. &#40;2020&#41;. <strong>Transcripts&#39; evolutionary history and structural dynamics give mechanistic insights into the functional diversity of the JNK family.</strong> <em>Journal of Molecular Biology</em> <a href="https://doi.org/10.1016/j.jmb.2020.01.032"><em>link</em></a></p>
</li>
<li><p>Ochoa, S., Mart√≠nez‚ÄêP√©rez, E., Zea, D. J., Molina‚ÄêVila, M. A., &amp; Marino‚ÄêBuslje, C. &#40;2019&#41;. <strong>Comutation and exclusion analysis in human tumors: A tool for cancer biology studies and for rational selection of multitargeted therapeutic approaches.</strong> <em>Human mutation, 40&#40;4&#41;, 413-425.</em> <a href="https://doi.org/10.1002/humu.23705"><em>link</em></a></p>
</li>
<li><p>Zea, D. J., Monzon, A. M., Parisi, G., &amp; Marino-Buslje, C. &#40;2018&#41;. <strong>How is structural divergence related to evolutionary information?.</strong> <em>Molecular phylogenetics and evolution</em>, 127, 859-866. <a href="https://www.sciencedirect.com/science/article/pii/S1055790317306188"><em>link</em></a></p>
</li>
<li><p>Marino-Buslje, C., Monzon, A. M., Zea, D. J., Fornasari, M. S., &amp; Parisi, G. &#40;2017&#41;. <strong>On the dynamical incompleteness of the Protein Data Bank.</strong> <em>Briefings in Bioinformatics</em> <a href="https://doi.org/10.1093/bib/bbx084"><em>link</em></a></p>
</li>
<li><p>Teppa, E., Zea, D. J., &amp; Marino‚ÄêBuslje, C. &#40;2017&#41;. <strong>Protein‚Äìprotein interactions leave evolutionary footprints: High molecular coevolution at the core of interfaces.</strong> <em>Protein Science</em>, 26&#40;12&#41;, 2438-2444. <a href="https://doi.org/10.1002/pro.3318"><em>link</em></a></p>
</li>
<li><p>Monzon, A. M., Zea, D. J., Marino-Buslje, C., &amp; Parisi, G. &#40;2017&#41;. <strong>Homology modeling in a dynamical world.</strong> <em>Protein Science</em>, 26&#40;11&#41;, 2195-2206. <a href="https://doi.org/10.1002/pro.3274"><em>link</em></a></p>
</li>
<li><p>Monzon, A. M., Zea, D. J., Fornasari, M. S., Salda√±o, T., Fernandez-Alberti, S., Tosatto, S. C., &amp; Parisi, G. &#40;2017&#41;. <strong>Conformational diversity analysis reveals three functional mechanisms in proteins.</strong> <em>PLOS Computational Biology</em>, 13, no. 2: e1005398. <a href="http://journals.plos.org/ploscompbiol/article?id&#61;10.1371/journal.pcbi.1005398"><em>link</em></a>  </p>
</li>
<li><p>Zea, D. J., Anfossi, D., Nielsen, M., &amp; Marino-Buslje, C. &#40;2016&#41;. <strong>MIToS. jl: mutual information tools for protein sequence analysis in the Julia language.</strong> <em>Bioinformatics</em>, btw646. <a href="https://academic.oup.com/bioinformatics/article-abstract/doi/10.1093/bioinformatics/btw646/2608634/MIToS-jl-mutual-information-tools-for-protein"><em>link</em></a></p>
</li>
<li><p>Zea, D. J., Monzon, A. M., Gonzalez, C., Fornasari, M. S., Tosatto, S. C., &amp; Parisi, G. &#40;2016&#41;. <strong>Disorder transitions and conformational diversity cooperatively modulate biological function in proteins.</strong> <em>Protein Science</em>, 25&#40;6&#41;, 1138-1146. <a href="http://onlinelibrary.wiley.com/doi/10.1002/pro.2931/full"><em>link</em></a></p>
</li>
<li><p>Iserte, J., Simonetti, F. L., Zea, D. J., Teppa, E., &amp; Marino-Buslje, C. &#40;2015&#41;. <strong>I-COMS: interprotein-COrrelated mutations server.</strong>  <em>Nucleic acids research</em>, 43&#40;W1&#41;, W320-W325. <a href="https://academic.oup.com/nar/article/43/W1/W320/2468009/I-COMS-Interprotein-COrrelated-Mutations-Server"><em>link</em></a></p>
</li>
<li><p>Parisi, G., Zea, D. J., Monzon, A. M., &amp; Marino-Buslje, C. &#40;2015&#41;. <strong>Conformational diversity and the emergence of sequence signatures during evolution.</strong> <em>Current opinion in structural biology</em>, 32, 58-65. <a href="http://www.sciencedirect.com/science/article/pii/S0959440X15000147"><em>link</em></a></p>
</li>
<li><p>Zea, D. J., Monzon, A. M., Fornasari, M. S., Marino-Buslje, C., &amp; Parisi, G. &#40;2013&#41;. <strong>Protein conformational diversity correlates with evolutionary rate.</strong> <em>Molecular biology and evolution</em>, 30&#40;7&#41;, 1500-1503. <a href="http://mbe.oxfordjournals.org/content/30/7/1500.full"><em>link</em></a></p>
</li>
</ul>
<h3 id="other_articles"><a href="#other_articles" class="header-anchor">üìÑ Other Articles</a></h3>
<ul>
<li><p>Parra, R. G., Defelipe, L. A., Guzovsky, A. B., Monzon, A. M., Cravero, F., Mancini, E., ... &amp; Gonzalez, N. N. &#40;2016&#41;. <strong>Highlights of the 1st Argentine Symposium of Young Bioinformatics Researchers &#40;1SAJIB&#41; organized by the ISCB RSG-Argentina.</strong> <em>PeerJ Preprints</em>, 4, e2494v1.  <a href="https://peerj.com/preprints/2494/"><em>link</em></a></p>
</li>
</ul>
<h3 id="book_chapters"><a href="#book_chapters" class="header-anchor">üìö Book chapters</a></h3>
<ul>
<li><p>Zea, D. J., Teppa, E., &amp; Marino-Buslje, C. &#40;2023&#41;. <strong>Easy Not Easy: Comparative Modeling with High-Sequence Identity Templates.</strong> In Homology Modeling: Methods and Protocols &#40;pp. 83‚Äì100&#41;. Springer US New York, NY. <a href="https://link.springer.com/protocol/10.1007/978-1-0716-2974-1_5"><em>link</em></a></p>
</li>
<li><p>Teppa, E., Zea, D. J., &amp; Buslje, C. M.  &#40;2012&#41;. <strong>Identification of Coevolving Amino Acids using Mutual Information.</strong> <em>iConcept Press</em>. <a href="https://www.researchgate.net/profile/Elin_Teppa/publication/261995698_Identification_of_Coevolving_Amino_Acids_using_Mutual_Information/links/00b4953631b3666180000000.pdf"><em>link</em></a></p>
</li>
<li><p>Monzon, A. M., Fornasari, M. S., Zea, D. J., &amp; Parisi, G. &#40;2019&#41;. <strong>Exploring Protein Conformational Diversity.</strong> <em>In Computational Methods in Protein Evolution</em> &#40;pp. 353-365&#41;. Humana Press, New York, NY. <a href="https://link.springer.com/protocol/10.1007/978-1-4939-8736-8_20"><em>link</em></a></p>
</li>
</ul>
<div class="page-foot">
    <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA 4.0</a> Diego Javier Zea. Last modified: February 21, 2024.
    Website built with <a href="https://github.com/tlienart/Franklin.jl">Franklin.jl</a> and the <a href="https://julialang.org">Julia programming language</a>.
</div>
</div></div> <!-- end of class container -->

<footer class="container">
  <p class="float-end"><a href="#">Back to top</a></p>
  <div class="footer-icons">
      <ul class="social-icons">
          <li><strong>Follow:</strong></li>
          <li><a href="https://twitter.com/diegojavierzea " rel="nofollow noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-twitter" viewBox="0 0 16 16">
              <path d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"/>
          </svg> X (Twitter)</a></li>
          <li><a href="https://github.com/diegozea" rel="nofollow noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-github" viewBox="0 0 16 16">
              <path d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.012 8.012 0 0 0 16 8c0-4.42-3.58-8-8-8z"/>
            </svg> GitHub</a></li>
      </ul>
  </div>
  <!--
  <p class="copyright">&copy; 2023 ‚Äî {{author}}</p>
  -->
</footer>

<!-- CONTENT ENDS HERE -->
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>


<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3"></script>
</body>
</html>
